tests/sentry/tasks/test_reports.py

```diff
@@ -168,47 +166,6 @@ def make_report(aggregates):
     assert has_valid_aggregates(interval, (project, make_report([1, 0, 0, 0]))) is True
 
 
-def test_percentiles():
```

Comment:
I think these were deleted because the related functions were deleted as they were unused.